src/creatures/creature.hpp

```diff
@@ -281,6 +281,10 @@ class Creature : virtual public Thing, public SharedObject {
 	const Outfit_t getDefaultOutfit() const {
 		return defaultOutfit;
 	}
+	bool isSupportOutfit() const {
```

Comment:
I  think that the prefix has will fit better than the is.
```suggestion
	bool hasSupportOutfit() const {
```